.github/workflows/offline_inference.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - uses: actions/checkout@v4
     - name: Run offline inference example
       run: |
-        nvidia-docker run --rm -t --net host --ipc host \
```

Comment:
why

Reply:
remove "--net host" 
1. Docker uses an isolated network setup to avoid the impact of a dirty Ray environment for each test.
2. Docker has no network communication requirements with the host, so there is no performance impact.